spark-etl/src/main/scala/geotrellis/spark/etl/EtlJob.scala

```diff
@@ -0,0 +1,15 @@
+package geotrellis.spark.etl
+
+import geotrellis.spark.etl.config._
+
+case class EtlJob(conf: EtlConf) extends Serializable {
```

Comment:
Once backends generate their own params, what is the purpose of this class? We could just pass EtlConf at that point, but does that lose meaning?